spec/whois/parsers/responses/whois.audns.net.au/au/status_registered_spec.rb

```diff
@@ -70,7 +70,7 @@
   describe "#registrar" do
     it do
       expect(subject.registrar).to be_a(Whois::Parser::Registrar)
-      expect(subject.registrar.id).to eq("MARKMONITOR")
+      expect(subject.registrar.id).to eq("MarkMonitor Inc.") # Registrar ID is no longer returned, use the name instead
```

Comment:
Please note that this file is auto-generated, you can't edit it directly. See the notice at the top of the file.